MapboxDirections/MBRouteStep.swift

```diff
@@ -557,7 +557,7 @@ open class RouteStep: NSObject, NSSecureCoding {
             return nil
         }
         maneuverType = ManeuverType(description: maneuverTypeDescription)
-        let maneuverDirectionDescription = decoder.decodeObject(of: NSString.self, forKey: "maneuverDirection") as! String
+        let maneuverDirectionDescription = decoder.decodeObject(of: NSString.self, forKey: "maneuverDirection")! as String
```

Comment:
Huh, how’d this `!` get here in the first place? This line should read:

```swift
guard let maneuverDirectionDescription = … as String? else { return nil }
```

An `init?(coder:)` method should be able to fail gracefully (hence the `?`), because it could be passed absolutely any object for perfectly legitimate reasons.